src/tests/ftest/pool/create_query.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/python
+"""
+(C) Copyright 2021-2022 Intel Corporation.
+
+SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+from pool_test_base import PoolTestBase
+
+
+class PoolCreateQueryTests(PoolTestBase):
+    # pylint: disable=too-many-ancestors
+    """Pool create tests.
+
+    All of the tests verify pool create performance with 7 servers and 1 client.
```

Comment:
Minor - According to the config, it's just 4 servers. And not really a "client" since it's all dmg.
```suggestion
    All of the tests verify pool create performance with 4 servers.
```
Also, is this checking "performance" in some way? (Maybe just copy-pasted)

Reply:
Copy paste is the direct way to hell :'(

- [ ] Fix hardware description